### PR TITLE
feat: ratings and modal component added

### PR DESCRIPTION
**Pull request by vrx29**
This PR contains code for Modal and Ratings component.

It will show 17 files, It is because I have created html page for every component and I have to put these new components in side navigation of all those previous components. 
So please have a look at modal.html and ratings.html and their respective CSS files.

Live Preview of Modal : https://gimesto.netlify.app/components/modal.html
Live Preview of Ratings : https://gimesto.netlify.app/components/rating.html

Screenshot 2022-02-20 at 7 49 06 PM
Screenshot 2022-02-20 at 7 49 14 PM

### CODE REVIEW

File: components/modal.html [hunk 1]
```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Gimesto</title>
+    <link rel="icon" type="image/png" sizes="32x32" href="../assets/favicon/favicon-32x32.png">
+    <link rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/atom-one-dark.min.css">
```
Comment (MayankKumar10) [inline components/modal.html:10]: Define this css files in root directory and import root directory here

**Review by foolhardy21 — APPROVED** [review]
code and ui look clean. nothing wrong stands out. good to merge.

File: styles/components/modal.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
+.modal-cont {
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
```
Comment (MayankKumar10) [inline styles/components/modal.css:5]: height by default is 100%, and try to define top 0 and left 0 in HTML tag so it will be inherited by default.

**Review by MayankKumar10 — APPROVED** [review]
Look into these suggestions everything else looks good to me.

File: styles/components/modal.css [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  padding-top: 8rem;
+  position: fixed;
+  z-index: 10;
+  background-color: #00000066;
```
Comment (jalajcodes) [inline styles/components/modal.css]: Use a variable here, as you are using in other places

**Review by jalajcodes — APPROVED** [review]
Looks good. Nice UI, you can merge it.

File: components/modal.html (hunk 1 above)
Comment (vrx29) [reply components/modal.html:10]: tried this before was having some issues thats why I am using it here.

File: styles/components/modal.css (hunk 2 above)
Comment (vrx29) [reply styles/components/modal.css:5]: This is for a particular modal component. that's why I have explicitly given the height as 100%.

File: styles/components/modal.css (hunk 3 above)
Comment (vrx29) [reply styles/components/modal.css]: sure will do this

**Pull request merged**